### PR TITLE
Add doc for Deploy API to Choreo Connect with Virtual Hosts

### PR DESCRIPTION
**Pull request by renuka-fernando**
## Purpose
PR for 4.0.0 branch
Same changes done for master branch https://github.com/wso2/docs-apim/pull/3841

### CODE REVIEW

File: en/docs/includes/deploy/assign-custom-hostname.md [hunk 1]
```diff
@@ -18,13 +18,13 @@
 
 3.  Click on the API to edit its configurations.
 
-     [![Edit the API](/../../../../assets/img/includes/deploy/select-api.png)](/../../../../assets/img/includes/deploy/select-api.png)
+     [![Edit the API](../../assets/img/includes/deploy/select-api.png)](../../assets/img/includes/deploy/select-api.png)
```
Comment (Mariangela) [inline en/docs/includes/deploy/assign-custom-hostname.md:21]: This change is wrong pls revert it.

File: en/docs/includes/deploy/assign-custom-hostname.md [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
 
-     [![Edit the API](/../../../../assets/img/includes/deploy/select-api.png)](/../../../../assets/img/includes/deploy/select-api.png)
+     [![Edit the API](../../assets/img/includes/deploy/select-api.png)](../../assets/img/includes/deploy/select-api.png)
 
 4.  Click **Deployments**.
 
 5.  Select the newly created Gateway environment, Virtual Host (foods.com), and a Revision.
 
-     [Create a new revision](/../../../../design/create-api/create-api-revisions) if no revisions exist.
+     [Create a new revision](../../design/create-api/create-api-revisions.md) if no revisions exist.
```
Comment (Mariangela) [inline en/docs/includes/deploy/assign-custom-hostname.md:27]: This change is wrong pls revert it.

**Review by Mariangela — CHANGES_REQUESTED** [review]

**Comment by Mariangela** [comment]
@renuka-fernando , what is the difference between this doc and the doc for the API GW? Is it just the configuration or steps as well?

File: en/docs/includes/deploy/assign-custom-hostname.md (hunk 1 above)
Comment (renuka-fernando) [reply en/docs/includes/deploy/assign-custom-hostname.md:21]: the image is not rendering in https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/deploy-api/exposing-apis-via-custom-hostnames/

**Comment by renuka-fernando** [comment]
> @renuka-fernando , what is the difference between this doc and the doc for the API GW? Is it just the configuration or steps as well?

This PR contains, steps for adding vhosts in MG.
End this also fix issues in vhosts doc for APIM as well.

File: en/docs/includes/deploy/assign-custom-hostname.md (hunk 1 above)
Comment (renuka-fernando) [reply en/docs/includes/deploy/assign-custom-hostname.md]: reverted with https://github.com/wso2/docs-apim/pull/3842/commits/013b5cef34ccd5cbfe5edccf52d59d8a67dbb6dd

File: en/docs/includes/deploy/assign-custom-hostname.md (hunk 2 above)
Comment (renuka-fernando) [reply en/docs/includes/deploy/assign-custom-hostname.md]: reverted with https://github.com/wso2/docs-apim/pull/3842/commits/013b5cef34ccd5cbfe5edccf52d59d8a67dbb6dd

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**